### PR TITLE
LPS-110733 Remove TinyMCE WYSIWYG Support

### PR DESCRIPTION
**Pull request by julien**

### CODE REVIEW

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:stable

**Comment by julien** [comment]
ci:test:relevant

**Comment by john-co** [comment]
ci:test:relevant

**Comment by julien** [comment]
ci:test:relevant

File: portal-impl/src/META-INF/javadocs-all.xml [hunk 1]
```diff
@@ -229204,6 +229204,9 @@ See https://issues.liferay.com/browse/LEP-2297.
 		</field>
 		<field>
 			<name>RESOURCE_TYPE_EDITOR_TINYMCEEDITOR</name>
+			<deprecated><![CDATA[As of 7.3.0, with no direct replacement.
+
+<p><i>This constant was originally used to define a editor type. In this version, this constant is never used and remains only to preserve binary compatibility. This constant will be permanently removed in a future version of the Java Servlet API.</i></p>]]></deprecated>
```
Comment (jbalsas) [inline portal-impl/src/META-INF/javadocs-all.xml]: "a editor type" -> "an editor type"

"a future version of the Java Servlet API" ... I doubt that's what's going on here. Please copy the message but adapt it to what you're actually doing.

**Comment by jbalsas** [comment]
We would also need an entry in the `BREAKING_CHANGES.md` file

**Comment by manoelcyreno** [comment]
Just started reviewing :)

**Comment by julien** [comment]
Re-submitted [here](https://github.com/julien/liferay-portal/pull/256)

**Comment by julien** [comment]
ci:stop

**Pull request closed without merging**